src/PrestaShopBundle/Resources/config/services/bundle/new_api.yml

```diff
@@ -50,3 +50,7 @@ services:
   PrestaShopBundle\Controller\Api\OAuth2\AccessTokenController:
     autowire: true
     autoconfigure: true
+
+  PrestaShopBundle\ApiPlatform\Converters\:
+    resource: "../../../../ApiPlatform/Converters/*"
```

Comment:
I would use this instead https://symfony.com/doc/current/service_container/tags.html#autoconfiguring-tags
It allows to add converters wherever you want (not restricted to a folder)